xapian-applications/omega/Makefile.am

```diff
@@ -76,10 +76,12 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/common \
 	-DCONFIGFILE_SYSTEM=\"$(sysconfdir)/omega.conf\" \
 	-DPKGLIBBINDIR=\"$(pkglibbindir)\"
+
 AM_CXXFLAGS += $(XAPIAN_CXXFLAGS)
 
 pkglibbindir = $(pkglibdir)/bin
 pkglibbin_PROGRAMS = omega
+
 dist_pkglibbin_SCRIPTS = mhtml2html outlookmsg2html rfc822tohtml vcard2text
```

Comment:
I guess these apparently randomly added blank lines are a result of adding and then removing things during development.  It's good practice to review the diff and eliminate extraneous changes like these.

Reply:
Yes, I didn't see that line.